### ISSUE TITLE
Link journal entries to people

### ISSUE THREAD

**Issue by jonasvp** [monicahq/monica#27]
Currently the "Journal" and "Activities" are separate. That seems rather artificial - how about merging them and allowing journal entries to link one or more people to them?

And for extra credit: allow linking people using Markdown syntax from the journal entries, e.g.

> I met [Mon](Monica Geller) on the street and talked to her about such-and-such. Then we met [Joey](Joey Tribbiani and the three of us had coffee.

**Comment by Nebucatnetzer** [comment]
This are as well two separate improvements. @djaiss can would it be helpful if I open issues for issues which contain more than one change? IMO this would help to get a better overview of what was already requested.

**Comment by djaiss** [comment]
@Nebucatnetzer Yes definitely. Ideally, one issue = one problem/bug/feature request, so it's easier to track.

**Comment by amitai** [comment]
Journal entries are now created automatically when you create an activity.
